src/libxtp/gwbse/bse.cc

```diff
@@ -655,5 +660,93 @@ BSE::Interaction BSE::Analyze_eh_interaction(const QMStateType& type,
   return analysis;
 }
 
+void BSE::Perturbative_DynamicalScreening(
+    const QMStateType& type, const Orbitals& orb,
```

Comment:
in general the reference to the paper would be nice

Reply:
Added